### ISSUE TITLE
Version 1.3.0 removing props

### ISSUE THREAD

**Issue by xbral** [Rocketseat/unform#41]
In version 1.3.0, it is removing the form's properties, like the Id, and returning only a data-testid, losing all the stylization done.

**Comment by cleitonper** [comment]
Aparently this issue was introduced in the commit [`a53d72b`](https://github.com/Rocketseat/unform/commit/a53d72b409cf6f1b886c8e60fe0fb13afce603fa#diff-9531e25902044f6bcd03b7d4b065fbc6), in the following lines:

```diff
-import React, { FormEvent, useState, FormHTMLAttributes } from "react"
+import React, { FormEvent, useState, useEffect } from "react";

-interface Props extends FormHTMLAttributes<HTMLFormElement> {
+interface Props {

-<form data-testid="form" {...rest} onSubmit={handleSubmit}>
+<form data-testid="form" onSubmit={handleSubmit}>
```

Removing `FormHTMLAttributes` we're losing all html attributes like `id` and `className` passed to `Form` component. In my case, with the removal of the `className` attribute from `form` element I lost all my *styled-components style*.

Currently i had to roolback to **v1.0.1** until this problem is not solved.

**Comment by diego3g** [comment]
For `classes` the #35 will solve. Other attributes were removed from form since we're overriding onSubmit function and TypeScript was complaining about that. We'll take a look at it and see if we can take all back.

**Comment by diego3g** [comment]
This issue will be solved by #77 and it will probably merge this week.
